### PR TITLE
Add Conference Program

### PR DESCRIPTION
**Pull request by dcwalk**
This addresses https://github.com/ournetworks/2018/issues/58

Yooooo! skeleton of how I imagine this working using data (srry, not srry), I think we can do like a pretty okay ICS build as a result, just thinking about isolating the "orga" events and also don't look too closely at uniqueIDs, etc....

screen shot 2018-06-01 at 11 28 31 pm
screen shot 2018-06-01 at 11 28 21 pm

### CODE REVIEW

**Comment by ana0** [comment]
Some stuff seems to be on here twice, is that on purpose/just a draft?

**Comment by dcwalk** [comment]
No that means i mucked up

File: _data/sessions.yml [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+  descriptionLink: "https://github.com/ournetworks/2018-submissions/issues/24"
+  sessionType: "talk"
+  theme: ""
+  date: 2018-07-14
+  startTime: ""
+  endTime: ""
+  length: 0.5
+
+- sessionID: 18242
+  title: "Our _stable_ networks"
```
Comment (ana0) [inline _data/sessions.yml]: I think this a duplicate?

File: _data/sessions.yml [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+  sessionType: "demo"
+  theme: ""
+  date: 2018-07-14
+  startTime: ""
+  endTime: ""
+  length:
+
+- sessionID: 18130
+  title: "Distributed web of care"
+  presenter: "Taeyoon Choi"
```
Comment (ana0) [inline _data/sessions.yml]: Also I think Taeyoon can't make it re: today's email :(

**Comment by dcwalk** [comment]
Thank you for the review @ana0 ! 
@garrying and @benhylau could we expedite a review on this so I could email out/post first draft on Saturday?

I think we can push this program than gradually fill in (handling the speakers section? links into description, etc... over time

In particular I've had to regig the schedule, actually feels really good for us ending at a good time both days

**Review by garrying — APPROVED** [review]
Added a style tweak to help with differentiating session blocks and c0d7def

**Pull request merged**